### PR TITLE
Remove workarounds for ninja/samurai bug

### PR DESCRIPTION
**Pull request by fingolfin**
This should in the final version skip ci & skip build, I think, but it seems prudent to first test if it works?

### CODE REVIEW

**Comment by fingolfin** [comment]
OK still getting errors -- I guess that means the new RootFS is not used or something like that? I guess I was a bit early ;-) sorry

File: G/GTK3/build_tarballs.jl [hunk 1]
```diff
@@ -16,10 +16,7 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir/gtk+-*/
 
-# Temporary workaround #1 issue with apk
-apk add samurai
-
-# Temporary workaround #2: `util-linux` has a file, `/usr/bin/wall`, owned by
+# Temporary workaround: `util-linux` has a file, `/usr/bin/wall`, owned by
 # `root:tty`, but `apk` can't chown the file to the `tty` group because it
```
Comment (giordano) [inline G/GTK3/build_tarballs.jl:20]: Also the installation of `apk-tools` can be removed, and the `--no-chown` flag to `apk` isn't necessary anymore (it's automatically passed by the new [`apk` wrapper](https://github.com/JuliaPackaging/Yggdrasil/blob/ee4e1f70bd631b5316c4b6860f15995d0b16e5b7/0_RootFS/Rootfs/bundled/utils/apk_wrapper.sh))

**Comment by giordano** [comment]
Thanks!

**Comment by fingolfin** [comment]
Shouldn't this have received a skip ci / skip build flag?

**Comment by giordano** [comment]
I did when [merging](https://github.com/JuliaPackaging/Yggdrasil/commit/11c1eb1f4ecc8ab9ad91f1b35d2942907cb31e33)

**Pull request merged**